### PR TITLE
Version 2.0! Make it translatable.

### PR DESCRIPTION
**Pull request by Jagget**
- Makes the mod translatable.
- Makes the mod updatable from other mods. (No need to built-in support for Unleveled Spells)
- Fixed the dark copy of the monster textures.

Screenshot 2024-01-07 001131
Screenshot 2024-01-07 130023
Screenshot 2024-01-07 130002

### CODE REVIEW

File: Settings/modsettings.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "Version": "1.5",
+    "Version": "2.0",
```
Comment (KABoissonneault) [inline Settings/modsettings.json]: I don't think you need to update the version if older settings are still compatible. You're causing users to reset their existing settings, when old settings are compatible with your changes.

Comment (Shapur1234) [reply Settings/modsettings.json]: On the other hand, if I were to download version "2.0" of a mod from nexus and install it, only to see as version "1.5" inside of DFU, I'd be pretty confused.

**Comment by Shapur1234** [comment]
Great work!
P. S.: If you have a Nexus account, I can add you as an author to the bestiary nexus pages, so you could upload patches directly, without needing to go through me.

File: Settings/modsettings.json (hunk 1 above)
Comment (Jagget) [reply Settings/modsettings.json]: That's the version of settings, not the mod. @KABoissonneault is most likely right. So, I changed it back

**Comment by Jagget** [comment]
> Great work! P. S.: If you have a Nexus account, I can add you as an author to the bestiary nexus pages, so you could upload patches directly, without needing to go through me.

Here I am. 
https://www.nexusmods.com/users/895825

This is my first C# project, so it might not be pretty or optimal, or bugless 👾

**Comment by Shapur1234** [comment]
> This is my first C# project, so it might not be pretty or optimal, or bugless 👾

This was my first C# project too (and my first "big" program) :)
If I were to rewrite this mod today, I would have done many things differently, like not creating my own custom (trash) format for storing text entries... so what you're working with definitely isn't optimal or pretty.

You should be now be able to edit the [Bestiary](https://www.nexusmods.com/daggerfallunity/mods/222) nexus pages.

File: Out/StandaloneLinux64/bestiary.dfmod.manifest [hunk 2]
```diff
@@ -18,27 +18,10 @@ Assets:
...
 - Assets/Game/Mods/Bestiary/Textures/button_sabertooth_tiger.png
-- Assets/Game/Mods/Bestiary/Resources/entry_fire_atronach.txt
-- Assets/Game/Mods/Bestiary/Resources/entry_daedroth.txt
-- Assets/Game/Mods/Bestiary/Resources/entry_rat.txt
-- Assets/Game/Mods/Bestiary/Resources/page_monsters2.txt
-- Assets/Game/Mods/Bestiary/Resources/entry_imp.txt
-- Assets/Game/Mods/Bestiary/Resources/entry_flesh_atronach.txt
-- Assets/Game/Mods/Bestiary/Resources/entry_gargoyle.txt
-- Assets/Game/Mods/Bestiary/Resources/entry_orc_sergeant.txt
 - Assets/Game/Mods/Bestiary/Textures/base_background.png
```
Comment (KABoissonneault) [inline Out/StandaloneLinux64/bestiary.dfmod.manifest:21]: Not sure why this PR got rubber stamped immediately. I can still find mistakes, like adding these unnecessary untracked assets in the .dfmod.json.
Haven't had time to have a look at the mod message

Comment (Jagget) [reply Out/StandaloneLinux64/bestiary.dfmod.manifest:21]: By all means keep looking and commenting! I'll open the new one if needed

I didn't publish anything yet.

Comment (Jagget) [reply Out/StandaloneLinux64/bestiary.dfmod.manifest:21]: Why `base_background.png` is unnecessary untracked asset?

Comment (KABoissonneault) [reply Out/StandaloneLinux64/bestiary.dfmod.manifest:21]: Sorry I meant to click on the right. There's a lot of files under a folder literally named Untracked. Not meant to be used by mods, this is a DFU output when packaging mods with source

**Comment by KABoissonneault** [comment]
For what it's worth, the mod messaging API seems fine. I'll have to test it some day

File: Out/StandaloneLinux64/bestiary.dfmod.manifest (hunk 2 above)
Comment (Jagget) [reply Out/StandaloneLinux64/bestiary.dfmod.manifest:21]: Oh that. It's a `.manifest` file. I clean it up in no time. The files are not included. Just records about them

Comment (KABoissonneault) [reply Out/StandaloneLinux64/bestiary.dfmod.manifest:21]: My bad, not used to people putting unnecessary generated files on the repo

Comment (Shapur1234) [reply Out/StandaloneLinux64/bestiary.dfmod.manifest:21]: > Not sure why this PR got rubber stamped immediately. I can still find mistakes, like adding these unnecessary untracked assets in the .dfmod.json. Haven't had time to have a look at the mod message

My bad, I only briefly skimmed through some of the changes and tried if the built version of the mod works. 
Thank you @KABoissonneault for looking through it and your advise!

**Pull request merged**